### PR TITLE
eslint on save

### PR DESCRIPTION
**Pull request by marcinel123**
Hi Adam,

Hope everything works with eslint.

Please check this on your computer and let me know.

Thanks,
Martin

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export const App = () => {
-  return <div className="text-lg">Ip-address-tracker!!</div>;
```
Comment (AdamKniec) [inline src/App.tsx]: this change does not seem to be related to EsLint. Please revert it

File: src/index.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { App } from "./App";
+import "./index.css"
```
Comment (AdamKniec) [inline src/index.tsx]: this change does not seem to be related to EsLint. Please revert it

File: package.json [hunk 3]
```diff
@@ -37,13 +37,15 @@
...
     "@typescript-eslint/eslint-plugin": "^5.54.0",
+    "autoprefixer": "^10.4.13",
     "eslint": "^8.35.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-config-standard-with-typescript": "^34.0.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-n": "^15.6.1",
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.32.2",
+    "postcss": "^8.4.21",
```
Comment (AdamKniec) [inline package.json]: this change does not seem to be related to EsLint. Please revert it

File: package.json [hunk 4]
```diff
@@ -37,13 +37,15 @@
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.54.0",
+    "autoprefixer": "^10.4.13",
```
Comment (AdamKniec) [inline package.json]: this change does not seem to be related to EsLint. Please revert it

**Review by AdamKniec — CHANGES_REQUESTED** [review]

**Review by AdamKniec — APPROVED** [review]
🎖️

**Comment by marcinel123** [comment]
Thanks for checking
